website/docs/r/cdn_frontdoor_origin_group.html.markdown

```diff
@@ -75,7 +75,12 @@ A `health_probe` block supports the following:
 
 * `path` - (Optional) Specifies the path relative to the origin that is used to determine the health of the origin. Defaults to `/`.
 
+-> **NOTE:** Health probes can only be disabled if there is a single enabled origin in a single enabled origin group.
+
+-> **NOTE:** To disable health probes you must remove them from your configuration files `azurerm_cdn_frontdoor_origin_group` resource definition.
```

Comment:
this is standard Terraform behaviour (omitting the field means it's removed/disabled), so we can remove this:

```suggestion
```

Reply:
I added this note because there seems to be some confusion between the Portal UI, where it is a check box. Since it is a check box many users are assuming that there is some sort of enable/disable field in the API, which there is not.

Fixed.